src/interfaces/history-item-interface.ts

```diff
@@ -1,72 +0,0 @@
-import { CorpTypeCd, FilingTypes } from '@/enums'
-import { CommentIF, DocumentIF, IsoDatePacific } from '@/interfaces'
-
-/**
- * A Filing History List item (ie, local object).
- * See also Api Filing interface.
- */
-export interface HistoryItemIF {
```

Comment:
This was a list item in the FHL, but now FHL uses the objects from the API (via a getter).

The "special" properties below are no longer on any object. Instead, they are getters in the components that need them (eg, IA component determines if completed, future effective, etc).